spec/models/assignment_spec.rb

```diff
@@ -2,13 +2,18 @@
 
 RSpec.describe Assignment, type: :model do
   i18n_scope = 'activerecord.errors.models.assignment.attributes'
-  let(:off) { FactoryGirl.build_stubbed(:offspring) }
-  let(:shift1) { FactoryGirl.build_stubbed(:shift) }
-  let(:user) { FactoryGirl.build_stubbed(:user) }
+  let(:offspring) { FactoryGirl.build(:offspring) }
```

Comment:
This creates a user, unless you create the user first and point your offspring to it